Containerfile.api

```diff
@@ -9,7 +9,7 @@ RUN go mod download
 COPY . .
 
 USER 0
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -buildvcs=false -o /planner-api cmd/planner-api/main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -buildvcs=false -o /planner-api cmd/planner-api/*.go
```

Comment:
Sorry that I didn't notice before, but wouldn't be better to use `cmd/planner-api/...` instead of `cmd/planner-api/*.go`

Reply:
It does not work for me.